### PR TITLE
Merged command-line options -p, -z, -e into one, -u --user-template.

### PR DESCRIPTION
**Pull request by BFH-ktt1**
This PR addresses #1021 and #1022

* #1021
* #1022

### CODE REVIEW

File: src/main/java/com/cburch/logisim/gui/start/Startup.java [hunk 1]
```diff
@@ -139,41 +139,35 @@ static void doPrint(File file) {
   // Non-used args short keys
   // --------------------------
   //    d    i k         u w t
```
Comment (MarcinOrlowski) [inline src/main/java/com/cburch/logisim/gui/start/Startup.java:141]: Can you update or remove this comment block?

File: src/main/java/com/cburch/logisim/gui/start/Startup.java [hunk 2]
```diff
@@ -606,13 +577,28 @@ private static RC handleArgTemplate(Startup startup, Option opt) {
...
-    if (errMsg == null && !startup.templFile.canRead()) errMsg = S.get("templateCannotReadError", file);
-    if (errMsg != null) return RC.WARN;  // FIXME: shouldn't we quit in such case?
-    return RC.OK;
+    // first we get the option
+    final var option = opt.getValue();
+    // we look if it is a file
+    final var file = new File(option);
+    if (file.exists()) {
+      startup.templFile = file;
+      String errMsg = null;
```
Comment (MarcinOrlowski) [inline src/main/java/com/cburch/logisim/gui/start/Startup.java:586]: `errMsg` is not used. Also the conditions can be merged and simplified:

```java
      if (!startup.templFile.canRead()) {
        logger.error(S.get("templateCannotReadError", file));
        return RC.QUIT;
      }
      return RC.OK;
```

File: src/main/resources/resources/logisim/strings/gui/gui_pl.properties [hunk 3]
```diff
@@ -714,12 +713,12 @@ argGeometryOption = Ustawia geometrię okna głóœneog głównego aplikacji
...
 # ==> argNoSplashOption =
 # ==> argTestVectorOption =
+# ==> argTestVectorOptionArgName =
 argOneTemplateError = Dozwolony jest tylko jeden szablon.
-argPlainOption = Używaj standardowego szablonu.
 argSubOption = Podczas ładowania zastąp plik1 biblioteki plikiem2
 argSubOptionArgName = plik
-argTemplateOption = Użyj pliku jako szablonu
-argTemplateOptionArgName = plik
+# ==> argTemplateOption =
```
Comment (MarcinOrlowski) [inline src/main/resources/resources/logisim/strings/gui/gui_pl.properties:720]:
```
argTemplateOption = Użyj pliku jako szablonu. Argument: plik
argTemplateOptionArgName = plik
```

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: src/main/java/com/cburch/logisim/gui/start/Startup.java (hunk 1 above)
Comment (BFH-ktt1) [reply src/main/java/com/cburch/logisim/gui/start/Startup.java:141]: Done

File: src/main/java/com/cburch/logisim/gui/start/Startup.java (hunk 2 above)
Comment (BFH-ktt1) [reply src/main/java/com/cburch/logisim/gui/start/Startup.java:586]: done

File: src/main/resources/resources/logisim/strings/gui/gui_pl.properties (hunk 3 above)
Comment (BFH-ktt1) [reply src/main/resources/resources/logisim/strings/gui/gui_pl.properties:720]: Thanks! done

File: src/main/resources/resources/logisim/strings/gui/gui.properties [hunk 4]
```diff
@@ -660,7 +660,7 @@ argSubOption = Substitutes library lib1 with lib2. Arguments: lib1 lib2.
 argSubOptionArgName = args
 argTemplateOption = Select template. Argument: "empty" for an empty template, "plain" for the default template, or the filename of the template to load.
 argTemplateOptionArgName = arg
-argTestCircGenOption = Open up the circ file and write a new one to later check the XML content. Arguments: <circ_input> <circ_output>
+argTestCircGenOption = Open a .circ file of any version of logisim and write a new one in the current format used in logisim-evolution. Arguments: <circ_input> <circ_output>
```
Comment (MarcinOrlowski) [inline src/main/resources/resources/logisim/strings/gui/gui.properties:663]: Shouldn't we spell `Logisim` with capital `L`? Also you used both `Logisim` and `Logisim-evolution`. Not sure if that can be confusing.

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: src/main/resources/resources/logisim/strings/gui/gui.properties (hunk 4 above)
Comment (BFH-ktt1) [reply src/main/resources/resources/logisim/strings/gui/gui.properties]: updated the l -> L. The confusion is minimal I think, so will not change that

**Review by MarcinOrlowski — APPROVED** [review]

**Pull request merged**